tests/ipsecondary_test.py

```diff
@@ -314,8 +282,10 @@ def decorator(test):
         @wraps(test)
         def wrapper(*args, **kwargs):
             repo_manager_exe_abs_path = path.abspath(repo_manager_exe)
-            with UptaneTestBackend(repo_manager_exe=repo_manager_exe_abs_path) as uptane_server:
-                result = test(*args, **kwargs, uptane_server=uptane_server)
+            with UptaneTestRepo(repo_manager_exe_abs_path) as repo, \
+                    FakeTestServerBackground(meta_path=repo.root_dir,
+                                             target_path=repo.target_dir) as uptane_server:
```

Comment:
Oh, you actually did it :)

Reply:
Hehe yes, I've changed some details to make it easier to adapt without the UptaneTestRepo class.